presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -119,6 +119,37 @@ public static Slice parseLine(@SqlType(StandardTypes.VARCHAR) Slice input)
         return serialize(geometry);
     }
 
+    @Description("Returns a LineString from an array of points")
+    @ScalarFunction("ST_LineString")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice stLineString(@SqlType("array(" + GEOMETRY_TYPE_NAME + ")") Block input)
+    {
+        if (input.getPositionCount() == 1) {
+            throw new PrestoException(INVALID_FUNCTION_ARGUMENT, "LineString cannot have one point");
+        }
+
+        MultiPath multipath = new Polyline();
+        for (int i = 0; i < input.getPositionCount(); i++) {
+            Slice slice = GEOMETRY.getSlice(input, i);
+            OGCGeometry geometry = deserialize(slice);
+            if (!(geometry instanceof OGCPoint)) {
+                throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("ST_LineString takes only an array of valid points, %s was passed", geometry.geometryType()));
+            }
+            OGCPoint point = (OGCPoint) geometry;
+            if (point.isEmpty()) {
+                throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("Input points to ST_LineString may not be empty, empty point at index %d", i));
```

Comment:
nit: perhaps, separate two sentences with a period: `Input points to ST_LineString may not be empty. Found empty point at index %d.`